src/Xamarin.Android.Build.Tasks/Xamarin.Android.EmbeddedResource.targets

```diff
@@ -56,6 +56,10 @@ This file is used by all project types, including binding projects.
       <Output TaskParameter="ResolvedEnvironmentFiles" ItemName="LibraryEnvironments" />
       <Output TaskParameter="ResolvedResourceDirectoryStamps" ItemName="_LibraryResourceDirectoryStamps" />
     </ReadLibraryProjectImportsCache>
+    <ItemGroup>
+      <FileWrites Include="@(ResolvedResourceDirectories->'$(Identity)..\res.zip')"
```

Comment:
Doesn't this assume/require that `%(Identity)` end with `\`?  What code *ensures* that's true?

As far as I can easily tell, `%(Identity)` will *never* end with `\`/`/`, as it appears to come from: https://github.com/xamarin/xamarin-android/blob/709be67be32b6734f74fafdb9c7b6c00421b1dee/src/Xamarin.Android.Build.Tasks/Tasks/ResolveLibraryProjectImports.cs#L195-L200

…and `Path.GetFullPath(dir)` *never* appends the directory separator char, unless `dir` already ends with it…